test/e2e/apps/containerrecreate.go

```diff
@@ -388,7 +389,8 @@ var _ = SIGDescribe("ContainerRecreateRequest", func() {
 							{Name: "app"},
 							{Name: "sidecar"},
 						},
-						ActiveDeadlineSeconds: utilpointer.Int64Ptr(3),
+						ActiveDeadlineSeconds:   pointer.Int64Ptr(3),
+						TTLSecondsAfterFinished: ptr.To[int32](3),
```

Comment:
Please either delete this line or set the value to more than 60 seconds. 
3 seconds is too short for the e2e test case to perform the necessary checks.

Reply:
@arya2004 please fix this error.